packages/babel-preset-react-app/create.js

```diff
@@ -100,7 +105,7 @@ module.exports = function (api, opts, env) {
           // Will use the native built-in instead of trying to polyfill
           // behavior for any plugins that require one.
           ...(!hasJsxRuntime ? { useBuiltIns: true } : {}),
-          runtime: opts.runtime || 'classic',
+          runtime: hasJsxRuntime ? 'automatic' : 'classic',
```

Comment:
is anything going to break by removing the option to pass in runtime?

Reply:
No, that was introduced for v4 and hasn't been released yet. @iansu can confirm though.

Reply:
Hmm. This doesn't look right. I think it is dangerous to check right in the preset. React might not necessarily be resolvable from that path (e.g. a monorepo setup). I think it was correct to keep this an option in the preset.

Reply:
It also makes the preset's behavior non-deterministic depending on whether you install React before or after you first compile you code. Wrong output might get cached.

Reply:
https://github.com/facebook/create-react-app/issues/9822

Reply:
Hi @gaearon, we discussed the risk around resolution on the last call, but weren't sure of the right path forward. I've asked @ianschmitz and @iansu for their thoughts on the new issue you've raised too - thanks!